docs/troubleshooting.md

```diff
@@ -176,7 +176,7 @@ With the modified configuration if you re-run the test above the log output shou
 2020-11-11T04:08:17.344Z	DEBUG	loggingexporter/logging_exporter.go:353	ResourceSpans #0
```

Comment:
Why this change?

Reply:
The coloring here is not a diff; only L179 is changing.

It looks to me like GitHub is highlighting that this code block is identified as `json` but is not valid JSON.

Reply:
The only change is `InstrumentationLIbrarySpans` -> `ScopeSpans`